modules/cnv_template_clone_DV.adoc

```diff
@@ -0,0 +1,22 @@
+[[template-DV-clone]]
+=== Template: DV Clone Configuration File 
+
+*example-clone-dv.yaml*
+
+----
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: "example-clone-dv"
+spec:
+  source:
+      pvc:
+        name: source-pvc
+        namespace: example-ns
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: "128Mi"
```

Comment:
I'd suggest to use 1G here as it's closer to the scale of sizes we'd actually be using.

Reply:
Fixed: https://github.com/openshift/openshift-docs/pull/12518/commits/8f994e43c293c237fe3c2192c2f6aa220c163b8c